packages/components/src/components/date-picker/_date-picker.scss

```diff
@@ -350,13 +351,13 @@
 
   .#{$prefix}--date-picker__day.inRange,
   .flatpickr-day.inRange {
-    background: $date-picker-in-range-background-color;
+    background: $highlight;
```

Comment:
Do you want to set `$highlight` to `$date-picker-in-range-background-color`, instead of directly setting `$highlight` here? This connects to the notion of component-level token discussed at #3502.

Reply:
That makes sense! I changed it back to $date-picker-in-range-color.